templates/cluster/hetzner-standalone-cp/templates/cluster.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: cluster.x-k8s.io/v1beta1
```

Comment:
Should not be a single file. It should be segregated into multiple files based on the CR that it represents. Please check other providers for examples.

Same for the hosted-cp chart as well.